### ISSUE TITLE
Chatbot keep getting error something went wrong

### ISSUE THREAD

**Issue by sha208** [neo4j-labs/llm-graph-builder#812]
Hi. Am trying to ask Q in chatbot, however it keeps giving error something went wrong.

![Screenshot 2024-10-21 154546](https://github.com/user-attachments/assets/a8f762f0-56b8-4854-8b34-a55b2862c9dc)

I am not using the gpt-4.0, so why did this issue happened. Is there any mistake from my configuration. Thanks in advance !

**Comment by kartikpersistent** [comment]
we changed the model name please take pull of latest DEV use the right config

**Comment by sha208** [comment]
After changing, am getting 'this connection don't have access, read-only mode". Please help on this.

**Comment by sha208** [comment]
![Screenshot 2024-10-22 123206](https://github.com/user-attachments/assets/abfad54f-fa02-42e6-984c-42791ce1109e)

**Comment by kartikpersistent** [comment]
do you have write access to your db?
can you share the backend env you are running

**Comment by sha208** [comment]
neo4j.conf -> dbms.databases.default_to_read_only=false

backend env in docker-compose.yml :

services:
  backend:
    build:
      context: ./backend
      dockerfile: Dockerfile
    volumes:
      - ./backend:/code
    environment:
      - NEO4J_URI=${NEO4J_URI-neo4j://database:7687}
      - NEO4J_PASSWORD=${NEO4J_PASSWORD-mypassword}
      - NEO4J_USERNAME=${NEO4J_USERNAME-neo4j}
      - OPENAI_API_KEY=${OPENAI_API_KEY-mykey}
      - DIFFBOT_API_KEY=${DIFFBOT_API_KEY-mykey}
      - EMBEDDING_MODEL=${EMBEDDING_MODEL-all-MiniLM-L6-v2}
      - LANGCHAIN_ENDPOINT=${LANGCHAIN_ENDPOINT-}
      - LANGCHAIN_TRACING_V2=${LANGCHAIN_TRACING_V2-}
      - LANGCHAIN_PROJECT=${LANGCHAIN_PROJECT-}
      - LANGCHAIN_API_KEY=${LANGCHAIN_API_KEY-}
      - KNN_MIN_SCORE=${KNN_MIN_SCORE-0.94}
      - IS_EMBEDDING=${IS_EMBEDDING-true}
      - GEMINI_ENABLED=${GEMINI_ENABLED-False}
      - GCP_LOG_METRICS_ENABLED=${GCP_LOG_METRICS_ENABLED-False}
      - UPDATE_GRAPH_CHUNKS_PROCESSED=${UPDATE_GRAPH_CHUNKS_PROCESSED-20}
      - NUMBER_OF_CHUNKS_TO_COMBINE=${NUMBER_OF_CHUNKS_TO_COMBINE-6}
      - ENTITY_EMBEDDING=${ENTITY_EMBEDDING-False}
      - GCS_FILE_CACHE=${GCS_FILE_CACHE-False}
#      - LLM_MODEL_CONFIG_anthropic_claude_35_sonnet=${LLM_MODEL_CONFIG_anthropic_claude_35_sonnet-}
#      - LLM_MODEL_CONFIG_fireworks_llama_v3_70b=${LLM_MODEL_CONFIG_fireworks_llama_v3_70b-}
#      - LLM_MODEL_CONFIG_azure_ai_gpt_4o=${LLM_MODEL_CONFIG_azure_ai_gpt_4o-}
#      - LLM_MODEL_CONFIG_azure_ai_gpt_35=${LLM_MODEL_CONFIG_azure_ai_gpt_35-}
#      - LLM_MODEL_CONFIG_groq_llama3_70b=${LLM_MODEL_CONFIG_groq_llama3_70b-}
#      - LLM_MODEL_CONFIG_bedrock_claude_3_5_sonnet=${LLM_MODEL_CONFIG_bedrock_claude_3_5_sonnet-}
#     - LLM_MODEL_CONFIG_fireworks_qwen_72b=${LLM_MODEL_CONFIG_fireworks_qwen_72b-}
      - LLM_MODEL_CONFIG_ollama_llama3=${LLM_MODEL_CONFIG_ollama_llama3-llama3,http://myip:8501}
    # env_file:
    #   - ./backend/.env
    container_name: backend
    extra_hosts:
      - host.docker.internal:host-gateway
    ports:
      - "8002:8002"
    networks:
      - net

**Comment by kartikpersistent** [comment]
@vasanthasaikalluri

**Comment by sha208** [comment]
Did i misconfigured something ?

**Comment by sha208** [comment]
update :

I tried connecting to neo4j aura, connection successful, however when i switch to my local instance, the same issue happened, "This user account does not have permission to access or manage data sources". Neo4j connection (Read only Mode).

**Comment by kartikpersistent** [comment]
yes neo4j aura is great to start. we will come with a way to support local running instance

**Comment by Harmannz** [comment]
> yes neo4j aura is great to start. we will come with a way to support local running instance

👋 Looking forward for the support for a locally running neo4j service. I've had to fallback to neo4j aura too.
